lib/lorekeeper/json_logger.rb

```diff
@@ -104,19 +110,15 @@ def remove_invalid_fields(fields)
       end
     end
 
-    def log_data(_severity, message)
+    def log_data(severity, message)
       # merging is slow, we do not want to merge with empty hash if possible
-      fields_to_log = if state[:extra_fields].empty?
-        state[:base_fields]
-      else
-        state[:base_fields].merge(state[:extra_fields])
-      end
+      fields_to_log = state[:extra_fields].empty? ? state[:base_fields] : state[:base_fields].merge(state[:extra_fields])
```

Comment:
To me the extended version with `if` is more legible

Reply:
not rubocop's point of view.
revert or leave as is?

Reply:
whatever you prefer

Reply:
I'll stick with rubocop's preference so it doesn't bug us every time.

Reply:
Could extract some of the logic into helpers methods to make it more readable

```rb
fields_to_log = extra_fields?(state) ? state[:base_fields].merge(state[:extra_fields]) : state[:base_fields]

.... 

def extra_fields?(state)
  state[:extra_fields].empty?
end
```

but that's kinda lame - moving all the logic into a helper method isn't bad:
```rb
fields_to_log = extract_state_fields(state)

...

def extract_state_fields(state)
  return state[:base_fields] if state[:extra_fields].empty? 

  state[:base_fields].merge(state[:extra_fields])
end
```